modules/conversion-center/widgets/contact-buttons.php

```diff
@@ -22,7 +22,7 @@ public function get_name(): string {
 	}
 
 	public function get_title(): string {
-		return esc_html__( 'Contact Buttons', 'elementor' );
+		return esc_html__( 'Single chat', 'elementor' );
```

Comment:
@mserino just a thought, if the naming is changing we may need to look at aria-labels associated with toggle and close buttons across variants

Reply:
@willhowat thanks, good call! I've updated all the occurrences with something more generic